.pre-commit-config.yaml

```
@@ -8,3 +8,9 @@ repos:
         files: ^pysqa/
       - id: ruff-format
         name: ruff format
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)